### PR TITLE
feat: integrate Shepherd tour with custom styling

### PR DESCRIPTION
**Pull request by sneha-4-22**
## Related Issue

fixes: #347 

## Description

https://github.com/Sanchitbajaj02/palettegram/assets/112711068/35666cfe-d206-4da0-a578-232f269f42e0

## Summary by CodeRabbit

- **New Features**
  - Introduced a guided tour feature using Shepherd.js to help users navigate the platform.
  - Added a "Start Tour" button in the navbar to initiate the guided tour.

- **Style**
  - Improved formatting of button styles for better visual consistency.

- **Documentation**
  - Updated styling rules for tour-related elements using Tailwind CSS.

### CODE REVIEW

File: src/components/core/navbar/styles.css
Comment (Sanchitbajaj02) [inline src/components/core/navbar/styles.css:1]: I prefer you to optimize this code. If possible try to use tailwind css

File: src/components/core/navbar/index.tsx [hunk 1]
```diff
@@ -18,7 +18,169 @@ import { getPosts } from "@/redux/reducers/postsReducer";
 import { saveBookmarkToStore } from "@/redux/reducers/bookmarkReducer";
 import { parseCookies } from "nookies";
 
+import Shepherd from 'shepherd.js'; 
+import '../../../../node_modules/shepherd.js/dist/cjs/css/shepherd.css';
+import './styles.css';
+
+
+const startTour = () => {
```
Comment (Sanchitbajaj02) [inline src/components/core/navbar/index.tsx:26]: It would be better if you create a separate file in utils for all the shepherd configuration

**Review by Sanchitbajaj02 — CHANGES_REQUESTED** [review]
Hi @sneha-4-22, Please do all the changes suggested by me and Coderabbit

File: src/components/core/navbar/index.tsx [hunk 2]
```diff
@@ -18,7 +18,11 @@ import { getPosts } from "@/redux/reducers/postsReducer";
 import { saveBookmarkToStore } from "@/redux/reducers/bookmarkReducer";
 import { parseCookies } from "nookies";
 
+import {startTour} from "../utils/startTour";
+
```
[the remark this answers is not in the document]
Comment (sneha-4-22) [reply src/components/core/navbar/index.tsx:22]: Shepherd.js does not support the type attribute for buttons directly

**Pull request merged**